ai-assistant/content.css

```diff
@@ -42,21 +42,18 @@
 .ck-content h4,
 .ck-content h5,
 .ck-content h6 {
-  line-height: 1.2em;
   padding-top: 0.8em;
   margin-bottom: 0.4em;
 }
 .ck-content blockquote,
 .ck-content ol,
 .ck-content p,
 .ck-content ul {
-  font-size: 1em;
-  line-height: 1.6em;
   padding-top: 0.2em;
   margin-bottom: var(--ck-spacing-large);
 }
 
 .ck-content {
   font-family: var(--font-family), sans-serif;
   line-height: 1.4;
-}
\ No newline at end of file
+}
```

Comment:
I would remove it since we've got default content styles. BTW `--font-family` variable is not defined.

Or set line-height using CSS variable:

```css
--ck-content-line-height: 1.4;
```